system-addon/content-src/components/Sections/Sections.jsx

```diff
@@ -2,11 +2,13 @@ const React = require("react");
 const {connect} = require("react-redux");
 const {injectIntl, FormattedMessage} = require("react-intl");
 const Card = require("content-src/components/Card/Card");
+const {PlaceholderCard} = Card;
 const Topics = require("content-src/components/Topics/Topics");
 const {actionCreators: ac} = require("common/Actions.jsm");
 
 const VISIBLE = "visible";
 const VISIBILITY_CHANGE_EVENT = "visibilitychange";
+const CARDS_PER_ROW = 3;
```

Comment:
Is there any way we can detect screen width and base this value on that? Otherwise this can happen. It's quite an edge case though...

![screen shot 2017-09-01 at 3 48 46 pm](https://user-images.githubusercontent.com/7688302/29985528-6b9d54ca-8f2d-11e7-8aa3-3964c58bd6f2.png)

Reply:
Interesting. Yes, this could be done, I think I'd like to file a follow-up for it though